### DISCUSSION TITLE
Whether there is any demo that is added to Disqus or Gitalk

### DISCUSSION THREAD

**Discussion by xxholly32** [vuejs/vitepress#3464] [Q&A]
Whether there is any demo that is added to Disqus or Gitalk

**Comment by bd-dxg** [comment]
我曾经实现过一次, 并且做了笔记,你可以看看,文章使用了简体中文,你可以使用翻译软件查看
I realized it once and took notes. You can take a look. The article is in simplified Chinese. You can use translation software to check it.
地址: https://bddxg.top/article/note/vitepress%E4%BC%98%E5%8C%96/%E7%BB%99vitepress%E6%B7%BB%E5%8A%A0%E8%AF%84%E8%AE%BA%E7%B3%BB%E7%BB%9F.html

请注意,目前我的网站已经关闭了Disqus,所以你看不到它是正常的
Please note that my website has shut down Disqus, so you can't see that it's normal.
